### ISSUE TITLE
[bug] code not rendering correctly

### ISSUE THREAD

**Issue by samkeen** [samkeen/nova-notes#56]
- [ ] Test is triple tick  sections render correctly on re-hydrate (hack it in the code).  I this works, we know we just need to make this replacement prior to delta->Markdown on save

in the editor three ticks is replaced with 

```html
<h2>Open a file</h2>
<p><br></p>
<div class="ql-code-block-container" spellcheck="false">
  <div class="ql-code-block">use std::fs::File;</div>
  <div class="ql-code-block">use std::io::ErrorKind;</div>
  <div class="ql-code-block"><br></div>
  <div class="ql-code-block">fn main() {</div>
  <div class="ql-code-block">    let f = File::open("hello.txt").unwrap_or_else(|error| {</div>
  <div class="ql-code-block">        if error.kind() == ErrorKind::NotFound {</div>
  <div class="ql-code-block">            File::create("hello.txt").unwrap_or_else(|error| {</div>
  <div class="ql-code-block">                panic!("Problem creating the file: {:?}", error);</div>
  <div class="ql-code-block">            })</div>
  <div class="ql-code-block">        } else {</div>
  <div class="ql-code-block">            panic!("Problem opening the file: {:?}", error);</div>
  <div class="ql-code-block">        }</div>
  <div class="ql-code-block">    });</div>
  <div class="ql-code-block"><br></div>
  <div class="ql-code-block">// Working with file f</div>
  <div class="ql-code-block"><br></div>
  <div class="ql-code-block">}</div>
</div>
```

After save and re-hydrate

we get
```html
<h2>Open a file</h2>
<p><br></p>
<p>use std::fs::File;</p>
<p>use std::io::ErrorKind;</p>
...
```

**Comment by samkeen** [comment]
the hack works

```javascript
// HACK a string with ```
        newContent = `
        this is code
        \`\`\`
        let x = "foo";
        \`\`\`
        this is not code
```
This renders as a code block in quill

**Comment by samkeen** [comment]
for this content

```markdown
## This is a code post

\```
let x = "hello";
println!(x)
\```
This is the end of the code post
```
You get these Quill deltas
```
ops: Array (7)
0 {insert: "This is a code post"}
1 {attributes: {header: 2}, insert: "↵"}
2 {insert: "↵let x = \"hello\";"}
3 {attributes: {code-block: "plain"}, insert: "↵"}
4 {insert: "println!(x)"}
5 {attributes: {code-block: "plain"}, insert: "↵"}
6 {insert: "↵This is the end of the code post↵"}
```

The Quill docs explain this format (need for attributes after content insert)
https://quilljs.com/docs/guides/designing-the-delta-format

**Comment by samkeen** [comment]
fixed via f00f948
